pkg/install/install.go

```diff
@@ -325,6 +333,65 @@ func (i *Installer) saveGraph(ctx context.Context, g graph) error {
 	return graph.CreateBlockBlobFromReader(bytes.NewReader([]byte(output)), nil)
 }
 
+func (i *Installer) createAndStoreGraph(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
+	clusterID := &installconfig.ClusterID{
+		UUID:    uuid.NewV4().String(),
+		InfraID: "aro",
+	}
+
+	g := graph{
+		reflect.TypeOf(installConfig): installConfig,
+		reflect.TypeOf(platformCreds): platformCreds,
+		reflect.TypeOf(image):         image,
+		reflect.TypeOf(clusterID):     clusterID,
+	}
+
+	i.log.Print("resolving graph")
+	for _, a := range targets.Cluster {
+		_, err := g.resolve(a)
+		if err != nil {
+			return err
+		}
+	}
+
+	adminInternalClient := g[reflect.TypeOf(&kubeconfig.AdminInternalClient{})].(*kubeconfig.AdminInternalClient)
+
+	ca := g[reflect.TypeOf(&tls.AdminKubeConfigSignerCertKey{})].(*tls.AdminKubeConfigSignerCertKey)
+	clientCertKey := g[reflect.TypeOf(&tls.AdminKubeConfigClientCertKey{})].(*tls.AdminKubeConfigClientCertKey)
+
+	err := i.generateNewClientKeyAndCert(ca, clientCertKey)
+	if err != nil {
+		return err
+	}
+
+	err = i.addKubeconfigContext(adminInternalClient, clientCertKey.CertRaw, clientCertKey.KeyRaw, "system:aro-service")
+	if err != nil {
+		return err
+	}
+
+	err = i.generateKubeconfig(adminInternalClient)
+	if err != nil {
+		return err
+	}
```

Comment:
pull these into a new separate install function which reads the graph, creates the kubeconfig, calls PatchWithLease to store it in a new database field.  Side advantage: we can use this function to retrofit the kubeconfig to already existing clusters later.